layouts/robots.txt

```diff
@@ -3,7 +3,5 @@ User-agent: *
 Disallow: /legacy/
 Disallow: /v1.0/
 Disallow: /v1.1/
-Disallow: /404/
-Disallow: /404.html
```

Comment:
This doesn't make sense to me. Why do we want these pages indexed?

Reply:
Because the /404 page got a backlink to the home page. it is still a useful page. The only page we are not supposed to index are: 
-Cart pages in e-commerce...
-The administration pages of the site 
-Categories and tags pages of the CMS (if not optimized)
-The PDFs identical to the pages of the site
- Pages from the internal engine
The pages " print " " send to "